src/Factory.php

```diff
@@ -142,6 +174,10 @@ protected function _newObject(string $className, array $ctorArgs): object
 
     protected function _factory($seed, $defaults = []): object
     {
+        if (is_object($defaults)) {
```

Comment:
see https://github.com/atk4/core/pull/252/files for changes in factory, not breaking...